PukingPredator/Assets/Scripts/Puking.cs

```diff
@@ -23,6 +23,11 @@ public class Puking : InputBehaviour
     /// </summary>
     private Vector3 pukeDirection => transform.forward + Vector3.up * 0.1f;
 
+    /// <summary>
+    /// If pukeForce is greater than this, the puke will also knock back items in front of the player
+    /// </summary>
+    private const float PUKE_EXPLODE_THRESH = 0.8f;
```

Comment:
i feel like we should probably reduce the threshold a bit, i might go as low as 50% cause right now it feels a bit excessive to hold it for 80% of 2 seconds given how hard you puke stuff by the time you've hit 50%

Reply:
ok, I reduced the threshold to 50%, but the boxcast will also be narrower(0.5-1 instead of always 1)